### PR TITLE
jenkins kube master/slave cleanup

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees ptal

### CODE REVIEW

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -358,23 +358,41 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
 plug-in] that allows Jenkins slaves to be dynamically provisioned on multiple
 container hosts using Kubernetes and {product-title}.
 
-The Jenkins image entrypoint also provides auto-discovery and auto-configuration
-of the Kubernetes plug-ins by scanning the project Jenkins is deployed in for
+To facilitate the using of the Kubernetes plug-in, {product-title} provides three
+additional images for you consumption.
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: "provides three images suitable for use as Jenkins slaves."

File: using_images/other_images/jenkins.adoc [hunk 2]
```diff
@@ -358,23 +358,41 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
...
+
+The first is https://github.com/openshift/jenkins/tree/master/slave-base[base image] for Jenkins slaves.
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: s/is/is a/

File: using_images/other_images/jenkins.adoc [hunk 3]
```diff
@@ -358,23 +358,41 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
...
+It pulls in both required tools (headless Java, the Jenkins JNLP client) and generally useful ones
+(including tar, zip, nss among others) as well as establishing the JNLP slave agent as the entrypoint.  And it provides
+Dockerfiles for both Centos and RHEL images.
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: "The image also includes the oc client tooling for invoking command line operations from within Jenkins jobs."

File: using_images/other_images/jenkins.adoc [hunk 4]
```diff
@@ -358,23 +358,41 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
...
 container hosts using Kubernetes and {product-title}.
 
-The Jenkins image entrypoint also provides auto-discovery and auto-configuration
-of the Kubernetes plug-ins by scanning the project Jenkins is deployed in for
+To facilitate the using of the Kubernetes plug-in, {product-title} provides three
+additional images for you consumption.
+
+The first is https://github.com/openshift/jenkins/tree/master/slave-base[base image] for Jenkins slaves.
+It pulls in both required tools (headless Java, the Jenkins JNLP client) and generally useful ones
+(including tar, zip, nss among others) as well as establishing the JNLP slave agent as the entrypoint.  And it provides
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: add git to that list.

File: using_images/other_images/jenkins.adoc [hunk 5]
```diff
@@ -358,23 +358,41 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
...
+Dockerfiles for both Centos and RHEL images.
+
+Two samples which extend this base image are provided:
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: s/are/are also/

File: using_images/other_images/jenkins.adoc [hunk 6]
```diff
@@ -358,23 +358,41 @@ https://wiki.jenkins-ci.org/display/JENKINS/Kubernetes+Plugin[Kubernetes
...
+
+The Jenkins image also provides auto-discovery and auto-configuration
+of slave images for the Kubernetes plug-ins by scanning the project Jenkins is deployed in for
 existing image streams with the label *role* set to *jenkins-slave*.
 
-When an image stream with this label is found, the entrypoint generates the
-corresponding Kubernetes plug-in configuration so you can assign your Jenkins
+When an image stream with this label is found, corresponding Kubernetes plug-in configuration
+is generated so you can assign your Jenkins
 jobs to run in a pod running the container image provided by the image stream.
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: "Note: this scanning is only performed once, when the Jenkins master is starting.  If you label additional imagestreams, the Jenkins master will need to be restarted to pick up the additional images."

**Review by bparees — DISMISSED** [review]

File: using_images/other_images/jenkins.adoc (hunk 1 above)
Comment (gabemontero) [inline using_images/other_images/jenkins.adoc]: part of next push

File: using_images/other_images/jenkins.adoc (hunk 2 above)
Comment (gabemontero) [inline using_images/other_images/jenkins.adoc]: part of next push

File: using_images/other_images/jenkins.adoc (hunk 4 above)
Comment (gabemontero) [inline using_images/other_images/jenkins.adoc]: part of next push

File: using_images/other_images/jenkins.adoc (hunk 3 above)
Comment (gabemontero) [inline using_images/other_images/jenkins.adoc]: part of next push

File: using_images/other_images/jenkins.adoc (hunk 5 above)
Comment (gabemontero) [inline using_images/other_images/jenkins.adoc]: part of next push

File: using_images/other_images/jenkins.adoc (hunk 6 above)
Comment (gabemontero) [inline using_images/other_images/jenkins.adoc]: part of next push

**Comment by gabemontero** [comment]
Updates pushed

**Comment by bparees** [comment]
lgtm

**Comment by gabemontero** [comment]
thx @bparees 

@openshift/team-documentation PTAL ... @bparees has signed off on technical content

**Comment by vikram-redhat** [comment]
@gaurav-nelson - PTAL. @bfallonf - can you please work with Gaurav? 

@gabemontero for Enterprise 3.3 or 3.4?

**Comment by gabemontero** [comment]
Technically the content is applicable to both.  But let's have @bparees
make the official declaration.

**Comment by bparees** [comment]
3.3 and 3.4, labels added.

**Comment by gabemontero** [comment]
ah ... online too ... thx @bparees

**Comment by gaurav-nelson** [comment]
Follow up #3288

**Comment by adellape** [comment]
@gaurav-nelson Looks like this was missing a milestone, but the follow-up had one, so I'm adding it here now since I'm doing the weekly publish.

Also adding the `dedicated` label to this and the follow-up PR.

**Pull request merged**